vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -944,14 +942,18 @@ void AdsImpl::CachePageScore(
   }
 }
 
+std::map<std::string, std::vector<double>> AdsImpl::GetPageScoreCache() const {
+  return page_score_cache_;
+}
```

Comment:
Do we need to return a copy of `page_score_cache_` or `const & ` is enough?

Reply:
Fixed